RAW/WebService.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  WebService.swift
+//  RAW
+//
+//  Created by Bharat malhotra on 13/07/18.
+//  Copyright © 2018 Bharat malhotra. All rights reserved.
+//
+
+import Foundation
+
+class WebService: NSObject {
+    
+    enum JSONError: String, Error {
+        case NoData = "ERROR: no data"
+        case ConversionFailed = "ERROR: conversion from JSON failed"
+    }
+    
+    func fetchRawData(sucess:@escaping(_ userData: [RawModel]) -> Void, failure: @escaping(_ failuremessage: String) -> Void){
+        guard let endpoint = URL.init(string:ServerConstants.APIEndPoint) else {
```

Comment:
can we write the signature in a better way? Perhaps only one closure that also returns the error?